sdk/monitor/azure-monitor-opentelemetry/azure/monitor/opentelemetry/_diagnostics/diagnostic_logging.py

```diff
@@ -32,19 +32,23 @@
 _DIAGNOSTIC_LOG_PATH = _get_log_path()
 
 _DISTRO_DETECTS_ATTACH = "4100"
-_INFO = "4101"
 _INSTRUMENTATION_SKIPPED = "4101"
+_INFO = "4102"
 
 _ATTACH_SUCCESS_DISTRO = "4200"
 _ATTACH_SUCCESS_CONFIGURATOR = "4201"
 _INSTRUMENTATION_SUCCEEDED = "4202"
 
 _DEPENDENCY_OVERLAP = "4300"
+_BACKOFF_EXPORTER = "4301"
```

Comment:
I see you're adding new ids. Are the corresponding ids being sent in any diagnostic logs yet? Or will that be implemented in another PR?